adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/CassandraFactory.java

```diff
@@ -43,13 +45,16 @@ public CassandraFactory(DnsResolver dnsResolver, String sidecarVersion)
     /**
      * Returns a new adapter for Cassandra 4.0 clusters.
      *
-     * @param session   the session to the Cassandra database
-     * @param jmxClient the JMX client to connect to the Cassandra database
+     * @param session                     the session to the Cassandra database
+     * @param jmxClient                   the JMX client to connect to the Cassandra database
+     * @param localNativeTransportAddress the address and port on which this instance is configured to listen
```

Comment:
@frankgh  keeps making me do this - if we can pick one style or the other and stick to it I'm happy to not align or align - I tend to agree w/ @yifan-c here but don't have a strong opinion one way or another.  Git/GitHub are smart enough to be able to highlight that it's just a whitespace change when you do make those kinds of alignment changes, so I don't think it's a huge deal, but I also get the "fewer changes in a commit is good" vibe.

Reply:
this is the default behavior when applying code style to the file, so I'd prefer to use it since other developers might also apply code style this way.